.github/workflows/cmake_ubuntu.yml

```
@@ -86,9 +92,10 @@ jobs:
 
     - name: Build
       working-directory: ${{runner.workspace}}/build
-      run: |
-        cmake --build . # OOM if more threads
+      run: cmake --build . -j 2
 
     - name: Test
-      working-directory: ${{runner.workspace}}/build
-      run: ctest -j 2 --output-on-failure
+      working-directory: ${{runner.workspace}}
```

actionlint (1.7.4)

property "workspace" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)